mix.exs

```diff
@@ -58,7 +58,8 @@ defmodule NervesHubLink.MixProject do
   defp dialyzer() do
     [
       flags: [:race_conditions, :error_handling, :underspecs, :unmatched_returns],
-      plt_add_apps: [:atecc508a, :nerves_key, :nerves_key_pkcs11],
+      plt_add_apps: [:atecc508a, :nerves_key, :nerves_key_pkcs11, :inets, :slipstream],
```

Comment:
`:slipstream` should be pulled in automatically via the deps so it really shouldn't be needed here. I'm also surprised that `:inets` needed to be added. That seems like it should be in `:included_applications` so that it's definitely loaded/initialized at runtime before this.